Libraries/CO.CDP.GovUKNotify/Models/EmailNotificationResponse.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text.Json.Serialization;
+
+namespace GovukNotify.Models;
```

Comment:
Wrong namespace in relation to the file location.

Reply:
Done